src/prefect/agent/agent.py

```diff
@@ -170,10 +185,13 @@ def _register_agent(self) -> str:
 
         return agent_id
 
-    def start(self) -> None:
+    def start(self, _loop_intervals: dict = None) -> None:
         """
         The main entrypoint to the agent. This function loops and constantly polls for
         new flow runs to deploy
+
+        Args:
+            - _loop_intervals (dict, optional): Default loop_intervals override for unit tests.
```

Comment:
I might just make the docstring description `Exposed for testing only.`, since we really want to make it clear to users that this isn't a public parameter.

Reply:
I did mentioned unit tests in earlier doc string, but I think this is more concise. Updated.